css/main.css

```diff
@@ -325,3 +325,53 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-form {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+  gap: 58px;
+  border: 1px solid white;
+}
+
+.contact-form > h2 {
+  width: 327px;
+  height: 216px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-form > input {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #979493;
+  padding: 10px;
+  margin-top: 5px;
+  width: 100%;
+  border: none;
+  outline: none;
+  border-bottom: 1px solid red;
+}
+
+.contact-form > textarea {
+  width: 327px;
+  height: 114px;
+  padding: 10px;
+}
+
+.final-social {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 15px;
+}
```

Comment:
Kindly review your styles and user interactions with the form. Consider making it as close with the following  [FIGMA TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=sD191wuasAZqCrMf-0)